build/config/ios/ios_sim.py

```diff
@@ -100,8 +127,15 @@ def Main():
                       default='localhost:8080',
                       help='Sky server address.')
 
+  parser.add_argument('--ios_sim_path', dest='ios_sim_path', required=False,
+                      help='Path to your iOS Simulator executable. '
+                      'Not normally required.')
+
   subparsers = parser.add_subparsers()
 
+  launch_parser = subparsers.add_parser('boot', help='Boot iOS Simulator')
```

Comment:
I was testing with that.  I agree it's probably not needed as an explicit command.